test/collections/CollectionNQueens.chpl

```diff
@@ -0,0 +1,192 @@
+use Barrier;

```

Comment:
You'll want to update this to `Barriers`, assuming #7035 is getting merged before this.